internal/pkg/cli/dialog/use_template.go

```diff
@@ -185,6 +185,8 @@ func convertStrValue(value string, targetType string) (interface{}, error) {
 		}
 	case `string`:
 		return value, nil
+	case ``:
+		return value, nil
```

Comment:
Pri lokalnom testovani som narazil na to, ze pre `kind=password` tu pride prazdny `type`.
Tak to fixujem zatial takto, nech to moze produkt testovat, a doplnim na `inputs` testy.

Reply:
Huh, zajímavý

Reply:
je to preto, lebo `type` sa vyzaduje iba pre `kind=input` .... a `kind=password` je vlastne tiez input, len s `hidden=true`.